src/pages/playing-for-keeps-terms.md

```diff
@@ -0,0 +1,145 @@
+---
+template: legal-page
+path: /playing-for-keeps-terms
+title: Playing for Keeps Terms of Use
+date: Last Updated April 12, 2020
+---
+# Rules & Rewards
+
+Keep is committed to helping you succeed on this program and to building a robust decentralized network. To this end, they've set up a [Discord server](https://discord.gg/wYezN7v) that anyone can join, and [technical documentation](https://docs.keep.network/run-random-beacon.html) to help you run Keep's random beacon client and start staking on testnet.
+
+## **Reward Guidelines**
+
+## Current Game of Keeps
```

Comment:
/cc @mhluongo looks like even though this page is called Playing for Keeps, the legal bits still use “Game of Keeps”. Not sure how much we care.